### PR TITLE
Fixed getCourseSections and getStudentUser

### PR DESCRIPTION
**Pull request by ZacharyMoran**

### CODE REVIEW

File: API/API.php [hunk 1]
```diff
@@ -1552,7 +1554,7 @@ function getCourseSections($courseID)
 		//prepare query to protect from sql injection
 		$query = $sqlite->prepare("SELECT * FROM Section WHERE COURSE_ID=:courseID AND AVAILABILITY=:avail");
 		$query->bindParam(':courseID', $courseID);
-		$query->bindParam(':avail', 1);
+		$query->bindParam(':avail', "1");
```
Comment (roda0313) [inline API/API.php]: I think you had it right the first time since avail is a bit? I don't think this will work

Comment (ZacharyMoran) [inline API/API.php]: it was giving me a 500 server error currently
and the query string worked as you saw

Comment (roda0313) [inline API/API.php]: After testing it looks like actually either will work. SQLite is just amazing like that

Comment (ZacharyMoran) [inline API/API.php]: is it an issue of i had a integer just floating with no variable instance?

Comment (ZacharyMoran) [inline API/API.php]: but my point is that the first one does not work, like test it through the url because you get a 500 error

**Pull request merged**